Samples/Dinner/Sources/Onboarding/Dinner.swift

```diff
@@ -0,0 +1,90 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Distributed Tracing open source project
+//
+// Copyright (c) 2020-2023 Apple Inc. and the Swift Distributed Tracing project
+// authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift OpenTelemetry open source project
+//
+// Copyright (c) 2021 Moritz Lang and the Swift OpenTelemetry project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+
+import Tracing
+
+func makeDinner() async throws -> Meal {
+  try await InstrumentationSystem.tracer.withSpan("makeDinner") { _ in
+    await sleep(for: .milliseconds(200))
+
+    async let veggies = try chopVegetables()
+    async let meat = marinateMeat()
+    async let oven = preheatOven(temperature: 350)
+    // ...
+    return try await cook(veggies, meat, oven)
+  }
+}
+
+func chopVegetables() async throws -> [Vegetable] {
+  try await otelChopping1.tracer().withSpan("chopVegetables") { _ in
```

Comment:
I know I'm coming in late, but how come this is explicitly referencing `otelChopping1`?

Reply:
Just to mock a multi node trace in one process. I'll make an actual multi process example soon to complement this with more "normal" usage

Reply:
Oh, got it. That wasn't explicitly clear when looking at this example what it was actually demonstrating